contributors/devel/conformance-tests.md

```diff
@@ -0,0 +1,159 @@
+
+Conformance tests in the default profile should cover features that are:
+
+- GA (not alpha or beta APIs, nor deprecated features)
+- portable (not dependent on provider-specific capabilities or on the public internet)
+- not an optional feature (e.g. not policy enforcement)
+- non-privileged (neither root on nodes, network, nor cluster)
+- should be stable and run consistently
+- cannot skip providers (there should be no Skip like directives added to the test), 
+especially in the Nucleus or Application layers as described 
+[here](https://github.com/kubernetes/community/blob/master/contributors/devel/architectural-roadmap.md).
+- should work with default settings for all configuration parameters 
+(example: the default list of admission plugins should not have to be tweaked for passing conformance).
+
+For each Kubernetes release, a Conformance Document will be generated
+that lists all of the tests that comprise the conformance test suite, along
+with the formal specification of each test. For example conformance document for 
+1.9 can be found [here](https://github.com/cncf/k8s-conformance/blob/master/docs/KubeConformance-1.9.md).
+This document will help people understand what features are being tested without having to look through
+the testcase's code directly.
+
+As each new release of Kubernetes provides new functionality, the subset of
+tests necessary to demonstrate conformance grows with each release. Conformance
+is thus considered versioned, with the same backwards compatibility guarantees
+as laid out in [our versioning policy](/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew).
+Conformance tests for a given version should be run off of the release branch
+that corresponds to that version. Thus `v1.2` conformance tests would be run
+from the head of the `release-1.2` branch. eg:
+
+ - A v1.3 development cluster should pass v1.1, v1.2 conformance tests
+
+ - A v1.2 cluster should pass v1.1, v1.2 conformance tests
+
+ - A v1.1 cluster should pass v1.0, v1.1 conformance tests, and fail v1.2
+conformance tests
+
+Conformance tests are designed to be run with no cloud provider configured.
+Conformance tests can be run against clusters that have not been created with
+`hack/e2e.go`, just provide a kubeconfig with the appropriate endpoint and
+credentials.
+
+```sh
+# setup for conformance tests
+export KUBECONFIG=/path/to/kubeconfig
+export KUBERNETES_CONFORMANCE_TEST=y
+
+# run all conformance tests
+go run hack/e2e.go -- --provider=skeleton --test --test_args="--ginkgo.focus=\[Conformance\]"
+
+# run all parallel-safe conformance tests in parallel
+GINKGO_PARALLEL=y go run hack/e2e.go -- --provider=skeleton --test --test_args="--ginkgo.focus=\[Conformance\] --ginkgo.skip=\[Serial\]"
+
+# ... and finish up with remaining tests in serial
+go run hack/e2e.go -- --provider=skeleton --test --test_args="--ginkgo.focus=\[Serial\].*\[Conformance\]"
+```
+
+### Defining Conformance Subset
```

Comment:
I was hoping you would merge these into the bulleted list above, merging or removing dupes where appropriate

Reply:
I tried my best to add them and rearrange them under the bullet points above. My bad, the reason for a cut and paste was to not lose content, it kind of defined what a conformance test is not where as the bullet points mentioned necessary attribute of the test.

Fixed @tpepper recommendation. 
Please review and let me know if any recording and rearrangement is required.